### PR TITLE
Remove Xten from function name

### PR DESCRIPTION
**Pull request by vezzi**
this is a pretty safe pull request.. with the only problem that breaks everything :-D
@Galithil the re-naming is needed, I am currently working on taca to generalise the concept of Run and having specific inherited classes. Have a parser that is Xten specific in the name is driving me creasy

### CODE REVIEW

**Comment by vezzi** [comment]
@Galithil I modified tests so that now all nosetests are passed

**Comment by Galithil** [comment]
well, it is xten specific for a reason though ...
Even if the name of the demultiplexing dir changes, the path to certain files changes between the old bcl2fastq and the new one.
Samplesheet.csv : it's a convention that holds true for XTen, but not for multiple-length indexes we have on Hiseq. I also believe that the format is different and that the parser will fail on non Xten SS.

I can only use laneBarcode.htm and lane.htm in Xten runs too.

**Comment by vezzi** [comment]
THe idea is to start to use bcl2fastq2.17 asap. at that point (i hope) the various files will be identical

**Comment by vezzi** [comment]
@Galithil now SampleSheetParser works for HiSeq and MiSeq

**Comment by Galithil** [comment]
I guess i would work. Is that good to merge ?

**Comment by vezzi** [comment]
I tried it on a couple of runs (Xten, MiSeq and HiSeq) and I cannot see problems.
it is currently  used only by TACA but al names need to change, in particular the ones involved in the deliver page creation. I am modifying the analysis section so that one does not need to be changed.

**Comment by vezzi** [comment]
@Galithil @ewels @senthil10 time to review this.... can be merge at the same time with TACA one, and deployed on preproc1, 2 and b5

**Comment by vezzi** [comment]
@kate-v-stepanova this repo will be extremely useful for you in case we start to work on hugin

File: flowcell_parser/classes.py [hunk 1]
```diff
@@ -97,8 +99,8 @@ def create_db_obj(self):
         
 
 
-
-class XTenUndeterminedParser(object):
+"""
```
Comment (Galithil) [inline flowcell_parser/classes.py]: Should this be removed ?

**Comment by Galithil** [comment]
Just a few questions about the DemuxSummary files.
Are these things that are outputed by illumina ? Or by TACA ?

**Comment by vezzi** [comment]
bcl2fast2.17 outputs them... however there is an exception that is when I demux a NoIndex lane on HiSeq, in this case TACA creates this file.... but form the point of view of the parser this is ok, as the parser only expects a file in a certain place in a certain format....

**Comment by Galithil** [comment]
okay.

I just wanted to be sure that it was something we were expecting from
bcl2fastq. all fine, then.

File: flowcell_parser/classes.py (hunk 1 above)
Comment (vezzi) [inline flowcell_parser/classes.py]: yes, I just removed it

**Comment by vezzi** [comment]
@Galithil I removed the commented function.This can be merged but deployed carefully as the function names and object names have changed.

**Comment by vezzi** [comment]
@Galithil and also this can be merged now

**Comment by Galithil** [comment]
Rerunning travis just in case, it was stuck with flowcell parser.

**Pull request merged**